React/apis-get.md

```diff
@@ -1,26 +1,81 @@
 # Integrating with an API in React
 
 ## Learning Goals
+
 - Revisit `axios` as our API library
 - Integrate a get API request in react lifecycle method
 
 ### Setup
-So far, our pets app has been set up to pull static data from the `App` component and pass that into the `PetCollection` which handles the display. We're going to modify this to utilize the [pets API](https://petdibs.herokuapp.com/pets) instead.
+
+So far, our pets app has been set up to pull static data from the `App` component and pass that into the `PetList` which handles the display. We're going to modify this to utilize an API instead.  For the purposes of learning we're going to use the [json-server](https://github.com/typicode/json-server) package which can take a json file and generate a sample REST server based on it.
```

Comment:
I don't love the inconsistency with how the node exercise worked by using `json-server` here.  If we're going to change this I think we want to update there too.  (And any lessons that reference the pets API.)

Reply:
I've already added a card to do so.